ui/app/components/pipeline-editor.hbs

```diff
@@ -189,7 +189,7 @@
 {{/if}}
 {{#if this.isShowingEditConnectorModal}}
   <PipelineEditor::ConnectorModal
-    @newConnectorType={{this.newConnectorType}}
+    @newConnectorType={{this.selectedNode.type}}
```

Comment:
Would it be fairly straightforward to create a regression test for this scenario? I'm not sure how much mocking we already have in place for this to be emulated in an e.g. application test

What do you think?

Reply:
Yep we definitely could. Its a weird bug that is due to leftover state from opening the connector modal, so we just have to write a specific acceptance test for it. I can do it in a follow up 👍